aws/rust-runtime/aws-config/src/default_provider.rs

```diff
@@ -347,5 +444,129 @@ pub mod credentials {
                 .expect("creds should load");
             assert_eq!(creds.access_key_id(), "correct_key_secondary");
         }
+
+        fn fs_from_tuples(tuples: &[(&str, &str)]) -> Fs {
+            let mut fs = HashMap::new();
+            tuples.iter().for_each(|(k, v)| {
+                let k = (*k).to_owned();
+                let v = v.as_bytes().to_vec();
+                fs.insert(k, v);
+            });
+            Fs::from_map(fs)
+        }
```

Comment:
probably worth just adding this to `Fs`, maybe `from_slice` like we have for env?